P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <post xml:id="p5" type="comment" who="#u4" synch="#t005" replyTo="#p4">
+                <p>“Wenn Sie diesen Gruppen also “mangelnde Bildung“ attestieren wollen, so
+                    verwenden Sie bereits einen bestimmten, kulturgebundenen Bildungsbegriff.”</p>
+                <p>Ich hoffe doch, wir können beim Bildungsbegriff der Aufklärung bleiben. Wer das
+                    nicht möchte, hat die Wissenschaft verlassen.</p></post>
+        </egXML>
+    </exemplum>
+    <remarks xml:lang="en" versionDate="2021-09-07">
+        <p>Even though in some CMC genres (e.g., WhatsApp), the other parties are informed by an
+            automated alert about the fact that another party is currently composing a new post,
+            they cannot track the process of verbalization, i.e. how the written utterance emerges
+            in the entry form on the user interface of its author. It is not until the author
+            performs a ‘posting’ action (e.g., by hitting the ‘enter’ key or by activating a ‘send’
+            button with the mouse) that the result of the composition process — the post — is made
+            available for the other parties. From the perspective of its addressees/readers, a post
+            is a stretch of text that has been composed in advance. Different from spoken utterances
+            but similar to edited text, (1) the reader of a post does not have access to the
+            verbalization process and (2) the mental processing of a post is not possible until the
+            process of verbalization (the composition) has been finished and the posting action has
+            been performed by the author. Posts occur in a broad range of written CMC genres: as
+            user ‘messages’ in chats and WhatsApp dialogues, as SMS messages, as tweets in Twitter
+            timelines, as individual comments following a status update on Facebook pages, as posts
+            in forum threads, as contributions on Wikipedia talk pages or in the comments section of
+            a weblog. [Element adopted from the <ref target="#TEIcmc2012">DeRiK schema</ref>.
+            Aspects of the model adopted from the <ref target="#descmr">CoMeRe Schema</ref>]. The
+                <gi>post</gi> element is a member of the model class model.common (via model.CMC).
+            This allows for using (and combining) occurrences of <gi>post</gi>, <gi>u</gi>,
+                <gi>incident</gi>, and <gi>kinesic</gi>in one and the same <gi>div</gi> in order to
+            be able to represent the combined use of written and spoken utterances and nonverbal
+            acts on the GUI of multimodal CMC environments.</p>
+	    <p>The use of the <att>modality</att> attribute is
+	    recommended, but not required. Its value is either
+	    <val>written</val> or <val>spoken</val>. Because spoken
```

Comment:
Style: never start a sentence with "because". ;-)

Reply:
I simply disagree — your style guide is incorrect. (As evidence, we use “Because” at the beginning of a sentence roughly 50 times in the _Guidelines_, although some of those are in comments.)